pysrc/bytewax/connectors/bigquery/bigquery_output.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from google.cloud import bigquery
+
+from bytewax.outputs import ManualOutputConfig
+
+
+class BigqueryOutputConfig(ManualOutputConfig):
```

Comment:
Minor point, but call this `BigQueryOutputConfig` to make casing consistent? Then the casing matches `DynamoDBOutputConfig` (since it's not `DynamodbOutputConfig`). I guess this is only the second time we have an output config whose name is more than one word.